.github/workflows/run-synth-pipeline.yml

```diff
@@ -16,9 +16,7 @@ jobs:
       with:
         username: ${{ secrets.DOCKER_USERNAME }}
         password: ${{ secrets.DOCKER_PAC }}
-    - 
-      name: Build pipeline
-      run: docker build -f dockerfiles/quipp-dev.Dockerfile -t turinginst/quipp-env:latest .
     -
-      name: Run pipeline
-      run: docker run turinginst/quipp-env:latest make
+      name: Run pipeline with privbayes-adult
+  
+      run: docker run -v $GITHUB_WORKSPACE:/quipp-pipeline --workdir /quipp-pipeline turinginst/quipp-env:base make run-privbayes-adult
```

Comment:
Not sure if this is the latest version of the way we use the CI, might need to be corrected when merging to develop

Reply:
This is now the latests. Seems more sensible to mount the repo to the docker image otherwise you cant get the outputs easily